bika/lims/subscribers/objectmodified.py

```diff
@@ -32,6 +32,12 @@ def ObjectModifiedEventHandler(obj, event):
             reference_versions[obj.UID()] = version_id + 1
             target.reference_versions = reference_versions
 
+    elif obj.portal_type == 'AnalysisRequest':
+        mp = obj.manage_permission
+        # Manage Analyses / Attachment Removal
+        # https://github.com/senaite/senaite.core/issues/780
+        mp(permissions.DeleteObjects, ['Manager', 'LabManager', 'Owner'], 0)
```

Comment:
In this case this is actually handled by the field:
https://github.com/senaite/senaite.core/blob/master/bika/lims/browser/fields/aranalysesfield.py#L137
However, probably better to disallow that on permission level

Reply:
This does not make sense in the object modified handler, because the code is not executed on WF transition
I encountered that already that no modfication event is fired on WF transition
therefore this here: https://github.com/senaite/senaite.core/blob/master/bika/lims/api/__init__.py#L96
However, this code would be needed for this: https://github.com/bikalims/bika.lims/blob/master/bika/lims/subscribers/on_transition.py